test/e2e/bandwidth_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+// BenchmarkBandwidthIntraNode runs the benchmark of bandwidth between pods on same node.
+func BenchmarkBandwidthIntraNode(b *testing.B) {
+	withPerfTestSetup(func(data *TestData) {
+		b.StartTimer()
+
+		podDefA := createPerfTestPodDefinition("perftest-a", perftoolContainerName, perftoolImage)
+		_, err := data.clientset.CoreV1().Pods(testNamespace).Create(podDefA)
```

Comment:
nit: you can probably use this more concise style here:
```
if _, err := data.clientset.CoreV1().Pods(testNamespace).Create(podDefA); err != nil {
  // ...
}
```

Reply:
Done.